results/customer-portal/installer-platform.manual-instructions.txt

```diff
@@ -0,0 +1,3 @@
+1. Download the installer executable.
+
+2. Double-click the installer file to run it. 
```

Comment:
You think we need to tell people how to install unsigned content? (Accept the UAC warning.)